_deployment/tests/alexa.sh

```diff
@@ -9,8 +9,9 @@ check_rank () {
     exit 0
   fi
 
+  res=0
```

Comment:
I think something like `status`, `code` or `status_code` would better describe what the variable is for.
```suggestion
  status=0
```
```suggestion
  code=0
```
```suggestion
  status_code=0
```

Reply:
"status" it shall be!